src/lightning_app/components/multi_node.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Type
+
+from lightning_app import structures
+from lightning_app.core.flow import LightningFlow
+from lightning_app.core.work import LightningWork
+from lightning_app.utilities.packaging.cloud_compute import CloudCompute
+
+
+class MultiNode(LightningFlow):
+    def __init__(
+        self,
+        work_cls: Type["LightningWork"],
+        nodes: int,
+        cloud_compute: "CloudCompute",
+        *work_args: Any,
+        **work_kwargs: Any,
+    ) -> None:
+        """This component enables performing distributed multi-node multi-device training.
+
+        Example::
+
+            import torch
+
+            import lightning as L
+            from lightning.components import MultiNode
+
+            class AnyDistributedComponent(L.LightningWork):
+                def run(
+                    self,
+                    master_address: str,
+                    master_port: int,
```

Comment:
In Lightning Trainer, we have moved away from the word "master'. We use main_address, main_port.

Reply:
Done !

Reply:
yes! thanks for calling this out @awaelchli